tests/OmniSharp.Roslyn.CSharp.Tests/SignatureHelpFacts.cs

```diff
@@ -733,6 +733,40 @@ void M1(int a,int b,int c)
             Assert.Equal(4, actual.Signatures.Count());
         }
 
+        [Theory]
+        [InlineData("dummy.cs")]
+        [InlineData("dummy.csx")]
+        public async Task LiteralContextMethod(string filename)
+        {
+            const string source =
+@"class A
+{
+    static void M()
+    {
+        string three = 3.ToString($$);
+    }
+}";
+            var actual = await GetSignatureHelp(filename, source);
+            Assert.NotEmpty(actual.Signatures);
```

Comment:
It would be great if this could also assert that one of the intended signatures exists.

Reply:
This way we can avoid potential signatures being removed when they shouldn't have been.